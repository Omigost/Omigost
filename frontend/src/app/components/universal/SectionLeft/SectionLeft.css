@import "../../../css/Theme.less";

.SectionLeft {
  padding-top: 1vw;
  padding-bottom: 1vw;
  padding-left: 0;
  padding-right: 15vw;
}

.SectionLeftNoMargin {
  position: relative;
  left: -20vw;
}