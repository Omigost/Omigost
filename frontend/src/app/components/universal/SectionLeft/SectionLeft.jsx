import React from 'react'
import './SectionLeft.css'

class SectionLeft extends React.Component {
  render() {
   
    const { children, withoutMargin } = this.props;
   
    return (
      <div 
        styleName={
          (withoutMargin)?('SectionLeftNoMargin'):('SectionLeft')
        }
      >
        {children}
      </div>
    )
  }
}

export default SectionLeft