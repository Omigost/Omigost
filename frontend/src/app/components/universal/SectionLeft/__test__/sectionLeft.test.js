import React from 'react';
import renderer from 'react-test-renderer';

import SectionLeft from '../SectionLeft.jsx';

test('Snapshot test of SectionLeft', () => {
    const component = renderer.create(
        <SectionLeft>
            <div>A</div>
        </SectionLeft>
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});