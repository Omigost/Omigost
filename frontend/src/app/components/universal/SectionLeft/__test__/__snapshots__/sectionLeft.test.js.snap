// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot test of SectionLeft 1`] = `
<div
  styleName="SectionLeft"
>
  <div>
    A
  </div>
</div>
`;
